tests/installation/configure_dasd.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Activates multipath when multipath activation message appears
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use parent 'y2_installbase';
+use testapi;
+use strict;
+use warnings;
+
+sub run {
+    my $dasd_channel = "0.0.0150";
```

Comment:
We could use single quotes if not interpolation is needed, but there are not strict guidelines on that. Anyway given that we don't need to filter every time, which can just pass the value to the function, so we don't need any assignation here.